hms-api/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think prettier might clas directly with the eslint settings :thinking: would be good to just use one